addons/mail/models/mail_thread.py

```diff
@@ -2115,45 +2121,40 @@ def _message_get_auto_subscribe_fields(self, updated_fields, auto_follow_fields=
             Override this method if a custom behavior is needed about fields
             that automatically subscribe users.
         """
-        if auto_follow_fields is None:
-            auto_follow_fields = ['user_id']
-        user_field_lst = []
+        field_list = list()
```

Comment:
It's a list of field names, you can call it `fnames`. And `[]` is more conventional than `list()`.